docs/en/docs/sql-manual/sql-reference/Account-Management-Statements/GRANT.md

```diff
@@ -39,7 +39,7 @@ The GRANT command has the following functions:
 
 >Note that.
 >
->"Grant the specified role to the user" is not supported in the current version
+>"Grant specified roles to user" is supported in versions 2.0 and later
```

Comment:
```suggestion
>"Grant specified roles to user" is supported in versions 2.0 and later
> <version since"2.0">
>
> Grant specified roles to user
> 
> </version>
```

Reply:
Adding version tag here is hard to render. So I think it it ok to explain it in text